oe_corporate_site_info.install

```diff
@@ -15,15 +15,16 @@ use Drupal\Core\StringTranslation\TranslatableMarkup;
 /**
  * Implements hook_install().
  */
-function oe_corporate_site_info_install(): void {
+function oe_corporate_site_info_install($is_syncing): void {
   // Configure the RDF SKOS graphs.
-  // @TODO Update the following conditional expression and update the function's signature with the usage of $is_syncing beginning from core version 8.9.
```

Comment:
I've done as in https://github.com/openeuropa/oe_content/blob/2.x/modules/oe_content_person/oe_content_person.install#L16 - don't change config in install if installing from config.